pylibui/libui/box.py

```diff
@@ -35,33 +35,32 @@ def uiBoxAppend(box, child, stretchy):
     :return: None
     """
 
-    return clibui.uiBoxAppend(box, child, stretchy)
+    clibui.uiBoxAppend(box, child, stretchy)
 
 
-# - void uiBoxDelete(uiBox *b, uintmax_t index);
-def uiBoxDelete(*args):
+# - void uiBoxDelete(uiBox *b, int index);
```

Comment:
Oh, great! So whenever if you can change it, change it! 👍 
In the future, we may have to implement some script to update automatically those function signatures so that they are always correct and up-to-date..